spec/system/frontend/solidus_stripe/checkout_spec.rb

```diff
@@ -2,218 +2,302 @@
 
 require 'solidus_stripe_spec_helper'
 
-RSpec.describe "Checkout with Stripe", :js do
+RSpec.describe 'Checkout', :js do
```

Comment:
If we're matching the file name maybe we can go one step further and use "SolidusStripe Checkout"?
No big deal, but it's easier to understand where is the problem in case of failures.